src/com/company/Account.java

```diff
@@ -0,0 +1,28 @@
+package com.company;
+
+/**
+ * Created by michaelhilton on 2/18/16.
+ */
+
+public class Account {
+    double principal,rate; int daysActive,accountType;
+    public static final int STANDARD = 0, BUDGET=1,
+            PREMIUM=2, PREMIUM_PLUS = 3;
+
+    public static double calculateFee(Account[] accounts)
```

Comment:
It doesn't make sense to me that this method would take an array. Shouldn't we call the calculate fee for each object on itself.